cosmo_tester/resources/attributes.yaml

```diff
@@ -4,6 +4,10 @@ cloudify_username: admin
 cloudify_password: admin
 cloudify_tenant: default_tenant
 
+# If the latest image name is not set, the latest image name will be generated
+# based on the image name prefix and the currently installed CLI version
+# e.g. cloudify-manager-premium-4.2.dev1
+cloudify_manager_latest_image: ''
```

Comment:
I don't have any idea for a better name, so if you don't either, let's go with this one for now :)